src/js/dyna-rules.js

```diff
@@ -25,7 +25,7 @@
 
 /******************************************************************************/
 
-import '../lib/publicsuffixlist/publicsuffixlist.js';
+import publicSuffixList from '../lib/publicsuffixlist/publicsuffixlist.js';
 
 import globals from './globals.js';
```

Comment:
In this case, `globals` is still needed for `globals.cloud`, but this can be addressed later.

Reply:
We can replace `globals.could` with `self.cloud`, this is code for user interface so it can be safely assumed `self` is always present.